.github/workflows/docker-nightly-build.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image Nightly Build CI
+
+on:
+  push:
+    branches: [ cron_build ]
+  
+  schedule:
+    # run daily 0:00 
+    - cron: '0 0 * * *'
+
+env:
+  ORG: timescaledev
+
+jobs:
+
+  # Build ubuntu TimescaleDB images for both TSL and OSS code.
+  timescaledb:
+
+    name: PG${{ matrix.pg }}${{ matrix.oss }}
```

Comment:
Missing an `oss` in the matrix?

Reply:
will remove oss. Don't need oss nightly builds.